contributing/development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -425,6 +419,20 @@ variable like this:
 This way, we won't have to manually set the directory location
 each time we want to reference it.
 
+You also need to add ``$RISCV_TOOLCHAIN_PATH/bin`` to your user's PATH environment variable.
+
+::
+
+	export PATH="$RISCV_TOOLCHAIN_PATH/bin:$PATH"
```

Comment:
As you've mentioned in the warning below, exporting this into your PATH may break things. We shouldn't encourage users to write this command, even just in the terminal. Instead, we should override the variable just for the command, instead of exporting it into the PATH, like this:

```
PATH="$RISCV_TOOLCHAIN_PATH/bin:$PATH" scons arch=rv64 use_llvm=yes linker=mold ...
```

Adjusting the code block below to include this at the start would be welcome.